tools/storage/reap_worktrees.sh

```
@@ -0,0 +1,203 @@
+# Two classes are handled:
+#   PHANTOM — registered in .git/worktrees but the directory is gone. Only an admin record remains.
+#             Safe to prune; nothing on disk is lost.
+#   FINISHED — directory exists, tracked tree clean, no unique untracked/ignored data, and every
+#             commit is reachable from a surviving ref. Removed through git, never rm -rf.
+#
+# Usage:  reap_worktrees.sh            # DRY RUN (default — prints the plan, changes nothing)
+#         reap_worktrees.sh --apply    # execute the plan
+set -uo pipefail
+
+ROOT="$(git rev-parse --show-toplevel 2>/dev/null)" || { echo "not a git repo"; exit 1; }
+cd "$ROOT"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)